src/extension.ts

```diff
@@ -401,7 +401,9 @@ export function activate(context: vscode.ExtensionContext): CodedAPI {
 
 			vscode.workspace.onDidOpenTextDocument(coverageanal.populateCurrent, coverageanal, context.subscriptions);
 
-			vscode.workspace.findFiles("*.lst", "").then(files => {
+			const crp = config(null).get("coverageReportPath", ".");
+			const crpp = path.join(crp, "*.lst");

```

Comment:
I believe this is not a full solution as paths will still be resolved from the root path. However right now this could be improved by specifying a `**/*.lst` glob instead of the previous `*.lst` glob. There are file watches already registered which trigger on events, so there would also be an inconsistency with this setting if you generate coverage data while the editor is running instead of only when it's starting up.

Reply:
agree